### ISSUE TITLE
Bipolar mode changes display : Feature request

### ISSUE THREAD

**Issue by muziker** [pdesaulniers/wolf-shaper#129]
It looks as if some other wave shapers in bipolar mode have a center point at the zero crossing mark, and converts the unipolar mode graph into the positive part of the graph, and a linear graph onto the negative part. Right now, the graph is used as-is in bipolar mode when switched from unipolar mode. Maybe the bi/uni button should also include this behaviour of having a center zeroed keypoint when switched

**Comment by pdesaulniers** [comment]
Yes, I think that would make sense. However, instead of setting a linear function on the negative part, I would simply convert the unipolar graph into an equivalent bipolar graph. That way, toggling the uni/bi switch would not affect the output by itself.

Alternatively, we could modify unipolar mode so that it displays both the negative and positive part of the function simultaneously. Every modification to the graph on the positive side would be mirrored on the negative side, and vice versa. In bipolar mode, both sides of the graph would be editable independently. The downside of this approach is that it wastes a bit of screen space in unipolar mode, since half of the graph would always show duplicate information.

I believe @unfa proposed this feature a few years ago :stuck_out_tongue: Related issue: https://github.com/pdesaulniers/wolf-shaper/issues/68
